systemtest/src/test/java/io/strimzi/systemtest/tracing/TracingST.java

```diff
@@ -648,6 +787,14 @@ void testProducerConsumerMirrorMakerConnectStreamsService() {
         LOGGER.info("Deleting topic {} from CR", TOPIC_TARGET_NAME);
         cmdKubeClient().deleteByName("kafkatopic", TOPIC_TARGET_NAME);
         KafkaTopicUtils.waitForKafkaTopicDeletion(TOPIC_TARGET_NAME);
+
+        LOGGER.info("Deleting topic {} from CR", TOPIC_NAME + "-target");
```

Comment:
I think that these explicit deletions are not needed it should be deleted by ResourceManager.